src/native/fs/fs_napi.cpp

```diff
@@ -1387,6 +1467,18 @@ void
 fs_napi(Napi::Env env, Napi::Object exports)
 {
     auto exports_fs = Napi::Object::New(env);
+    uv_lib_t *lib = (uv_lib_t*) malloc(sizeof(uv_lib_t));
+
+    if(const char* env_p = std::getenv("GPFS_DL_PATH")) {
```

Comment:
I think this all if is also reminders of my old code. we should stay only with the new if.

Reply:
not sure what was needed to be resolved

Reply:
@jackyalbo Can you please go over your comments and see what can be resolved vs. what still requires fixing?

Reply:
in anycase, this change looks harmless, resolving.